### PR TITLE
Neon tracksuits.

### PR DESCRIPTION
**Pull request by Firefox13**
## About The Pull Request

This PR adds four new neon tracksuits that look hella cool. They are added in loadout.

## Why It's Good For The Game

Diversity and style.
изображение

## Changelog 
:cl: Firefox13, Jefferyjefferson
add: Neon tracksuits in four colours, available NOW in your loadout!
/:cl:

### CODE REVIEW

**Comment by jaffjafferson** [comment]
speedmerge this

**Comment by jaffjafferson** [comment]
speedmerge this

**Comment by Firefox13** [comment]
speedmerge this

**Comment by jaffjafferson** [comment]
he's right you know

**Comment by GrayRachnid** [comment]
how do these look with reinforced shoes, not jackboots.
Some newer sprites use jackboots as their lower cut off point which results in them having ankle space when using reinforced shoes.

**Comment by Firefox13** [comment]
> 
> 
> how do these look with reinforced shoes, not jackboots.
> Some newer sprites use jackboots as their lower cut off point which results in them having ankle space when using reinforced shoes.

those are reinforced shoes in preview image!

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -189,3 +189,28 @@ obj/item/clothing/under/leisure/pullover
...
+	name = "blue neon tracksuit"
+	desc = "A tracksuit lined with blue neon fibers. Slightly luminescent."
+	icon_state = "neonb"
+	item_state = "bl_suit"
+
+obj/item/clothing/under/neon/red
+	name = "red neon tracksuit"
+	desc = "A tracksuit lined with red neon fibers. Slightly luminescent."
+	icon_state = "neonr"
+	item_state = "bl_suit"
```
Comment (SpadesSlick) [inline code/modules/clothing/under/miscellaneous.dm:216]: Endline required in the end of the file.

**Pull request merged**